packages/terra-slide-panel/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```

Comment:
Since this issue was logged as a bug, would you update this heading to be `### Fixed`. 

Generally we follow this guide for our change-log entries: https://keepachangelog.com/en/1.0.0/

Reply:
[Fixed](https://github.com/cerner/terra-framework/pull/1071/commits/f13cab61cea812d92a6bcc163b0aa814be8ab3f5#diff-7e3ed4ed30d6886ce786592bdd93daf2R6)